Ginger/GingerCoreNET/RunLib/CLILib/CLIProcessor.cs

```diff
@@ -416,6 +417,25 @@ private async Task<int> HandleRunOptions(RunOptions runOptions)
                     Reporter.ToLog(eLogLevel.INFO, string.Format("Using provided ExecutionID '{0}'.", mCLIHelper.ExecutionId.ToString()));
                 }
             }
+            mCLIHelper.ReRunFailed = runOptions.ReRunFailed;
+            if (runOptions.ReRunFailed)
+            {
+                if(!string.IsNullOrEmpty(runOptions.ReferenceExecutionID))
+                {
+                    if (!Guid.TryParse(runOptions.ReferenceExecutionID, out Guid temp))
+                    {
+                        Reporter.ToLog(eLogLevel.ERROR, string.Format("The provided ExecutionID '{0}' is not valid.", runOptions.ReferenceExecutionID));
```

Comment:
Please change message to Provided Reference ExecutionId is not Valid

Reply:
Done